pygotham/frontend/templates/layouts/base.html

```diff
@@ -14,12 +14,13 @@
     <link rel="icon" href="{{ url_for('static', filename='img/favicon.png') }}">
 
     <script src="{{ url_for('static', filename='bower_components/modernizr/modernizr.js') }}"></script>
+    {% block extra_head %}{% endblock %}
   </head>
   <body class="{% block body_class %}{% endblock %}">
     <nav class="top-bar" data-topbar data-options="mobile_show_parent_link: false">
       <ul class="title-area">
         <li class="name">
-          <h1><a href="{{ url_for('home.index') }}">{{ current_event }}</a></h1>
+          <a href="{{ url_for('home.index') }}"><img src="{{ url_for('static', filename='img/logo-notext.svg') }}" alt="{{ current_event }} Home"></a>
```

Comment:
I think the `alt` should be `"{{ current_event }} logo"`. `"Home"` could then be the `title` on the anchor.